### ISSUE TITLE
Why use aug image as model input?

### ISSUE THREAD

**Issue by Duplex3345678** [kekmodel/MPL-pytorch#3]
Thank you for your implementation! Here are some confusions.

In your code, I think images_uw is unlabeled and origin image, and images_us is unlabeled and RandAugment image.

In paper v.3(https://arxiv.org/pdf/2003.10580v3.pdf) Algorithm1,  when "Update the student using the pseudo label", author use unlabeled and origin image to update StudentModel, but in your code, 
```sh
s_images = torch.cat((images_l, images_us))
s_logits = student_model(s_images)
.....
s_loss = criterion(s_logits_us, hard_pseudo_label)
```
Why you use images_us (RandAugment image) as the StudentModel input to compute s_loss instead of images_uw ?

Same questions in Algorithm1 about "Compute the teacher’s feedback coefficient as in Equation 12" and "Compute the teacher’s gradient from the student’s feedback", the author use unlabeled and origin image as the input, but in your code all unlabeled image input is RandAugment, not origin. Can I ask why?

Thank you!

**Comment by kekmodel** [comment]
https://github.com/google-research/google-research/issues/534#issuecomment-769559165

**Comment by Duplex3345678** [comment]
Thank for your support. I appreciate it!
